Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml

```diff
@@ -9,4 +9,5 @@
     Syringe: 5
     ClothingEyesHudMedical: 2
     ClothingEyesEyepatchHudMedical: 2
+    ClothingEyesMedMesons: 2
```

Comment:
I was kind of concerned of adding another set of medhuds to the vend, but I don’t think there was ever such an issue where too many people have a hud. 

I also think it would be kind of weird to have it be an uneven amount for each. I don’t know, maybe I’ll make med glasses to offset that lol

Reply:
I'm going to wait for a second opinion before I make changes (mostly because I'm not at my laptop).

I personally haven't experienced an issue where people just walk in and become a self appointed doctors (except the chaplain but that's its own problem).

if anything people just nab one during nuke-ops to target low HP nukies. Even then, I don't see people take advantage of that often.

Reply:
I never see a tide of people powergaming medhuds and every shift every vendor/locker has their extra medhuds leftover. Adding another source of extras isn't really going to change much in that department